src/main/java/core/basesyntax/service/ShopService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {
+    void processTransaction(List<FruitTransaction> fruitTransactions,
+                            Map<String,Integer> inventory);
```

Comment:
```suggestion
                            Map<String, Integer> inventory);
```
also pay attention to code style, alignment. Fix the same issue in all places. Use ctrl + alt + L (on windows or corresponding combination on other systems) to auto-align the file